src/openklant/components/contactgegevens/mixins.py

```diff
@@ -41,14 +43,13 @@ class AdresMixin(models.Model):
     adres_land = models.CharField(
         _("land"),
         help_text=_(
-            "Een code, opgenomen in Tabel 34, Landentabel, die het land (buiten Nederland) "
-            "aangeeft alwaar de ingeschrevene verblijft."
+            "ISO 3166-code die het land (buiten Nederland) aangeeft waar de abonnee woont."
```

Comment:
Not sure if the second part was changed on purpose?
```suggestion
            "ISO 3166-code die het land (buiten Nederland) aangeeft alwaar de ingeschrevene verblijft."
```